tests/containers/build-base

```diff
@@ -1,12 +1,12 @@
 FROM quay.io/centos/centos:stream9
 
-RUN dnf install -y dnf-plugin-config-manager
 # CRB is required for meson
-RUN dnf config-manager -y --set-enabled crb
 # EPEL is required for python3-flake8 and codespell
-RUN dnf install -y epel-release
 
-RUN dnf install \
+RUN dnf install -y dnf-plugin-config-manager && \
+    dnf config-manager -y --set-enabled crb && \
+    dnf install -y epel-release && \
+    dnf install --setopt=tsflags=nodocs \
```

Comment:
Could this be just `dnf install --nodocs`?

Reply:
You might want to add a dnf -y update \ to this prior to all of the others.

Reply:
yes, `--nodocs` should be an alias to that, updating.
Also adding the update command as suggested.

Reply:
added a second commit addressing above comments